### ISSUE TITLE
Issue formatting delegate event nodes

### ISSUE THREAD

**Issue by fpwong** [fpwong/BlueprintAssistWiki#35]
```
Hello,
Thanks for the amazing plugin! :grinning:
I am experiencing problems with sequences and events.
I also have the Blueprint assist plugin enabled with Auto Position Event nodes on.
When formatting a sequence, the comments are on top of each other (comment node padding is 30,30).
Each time I format, the event goes down, increasing the comment box and adding below nodes to the comment.
When I CTRL+Z, it becomes a big mess.

See it in video: https://youtu.be/sKH1S-msbUU

Is there a magical config to use to solve this problem?

It would be nice for example to set min distance between box (for not-child comments of course).
```

Video link: https://youtu.be/sKH1S-msbUU

**Comment by fpwong** [comment]
Hi Louspirit, I'm fairly certain this issue is not related to the comment boxes or the auto position event node setting. 

I did fix an issue similar to this recently. Try updating the plugin to version 3.0.0 and see if the issue is still there. 

Even though the bug is gone, your blueprint will still looks a bit ugly. There are a two methods which should fix this:

1. The *magical config*, enable `treat delegates as execution pins`
2. Instead of binding events via wire. Drag from the event pin and use the `Create Event` node.

**Comment by Louspirit** [comment]
Hello and thanks for replying.
**It is indeed much better !**
![image](https://user-images.githubusercontent.com/2952456/121500253-6c30eb00-c9de-11eb-9398-5a69f34b029d.png)

There is still a little comments overlapping, can we do something about it?
I would personally prefer to align the event node to its execution rather than its caller. Is it possible or _CreateEvent_ would be the only solution?

**Comment by fpwong** [comment]
I have an experimental feature which needs a lot of work (Account For Comment Nodes). Would not suggest using this yet but as there are issues right now but it does kinda work atm for simple cases. 

I am hoping to fix this the comment node setting by 3.1.0.

As for the event nodes, there is no other alternative right now. Personally I use CreateEvent and run Format All to align the other event nodes.
